go/vt/wrangler/vexec.go

```diff
@@ -323,6 +360,8 @@ func (wr *Wrangler) getWorkflowActionQuery(action string) (string, error) {
 		query = fmt.Sprintf(updateSQL, encodeString("Stopped"))
 	case "start":
 		query = fmt.Sprintf(updateSQL, encodeString("Running"))
+	case "update":
+		// We don't use the SQL interface, so this is only for dry-run purposes.
```

Comment:
Is there a way to validate that this is indeed a dry run? And return a `vterrors.Code_INTERNAL` if it isn't?